react/Header/NotificationsLink/NotificationsLink.less

```diff
@@ -0,0 +1,24 @@
+@import (reference) "~seek-style-guide/theme";
+@import (reference) "../Header.less";
+
+.bell {
+  padding: 15px 0;
+  position: relative;
+  top: 1px;
+  color: @sk-pink;
```

Comment:
How married to the pink are we for this? Apart from the contrast on mobile being a little challenging, I feel like it would look much more refined to follow the text colour, eg. black on desktop and white on mobile. Thoughts?

Reply:
100% agree, this was a design call made by Carmelia. I think since she's not in, let's just flip it to black on desktop, white on mobile.

Reply:
Confirmed with Carmelia this change is fine to roll with.